var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -604,6 +611,9 @@ def url_for_version(self, version):
     depends_on("eigen@3:", when="+machdyn")
     depends_on("py-cython", when="+mliap+python")
     depends_on("py-cython", when="+ml-iap+python")
```

Comment:
Cython should probably be a build-dep, there are no libs to link to

Reply:
Limited to `build` dependency.